### ISSUE TITLE
Resources side panel has spaces in links

### ISSUE THREAD

**Issue by gergoerdosi** [hapijs/hapijs.com#383]
```
http://hapijs.com/resources#Gists and code samples
http://hapijs.com/resources#Projects built with hapi.js
```

**Comment by nickfranciosi** [comment]
#411

**Comment by Marsup** [comment]
What's the problem with this exactly ? It seems to work for me, even with spaces.

**Comment by nickfranciosi** [comment]
I noticed that also, but thought that maybe since the HTML spec specifies no spaces in the id that the owners of this project had opened the issue so I decided to try to get a PR in. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

No worries if you don't need it merged in. Thanks!

**Comment by gergoerdosi** [comment]
@Marsup: It works, but think about copy-paste. When I insert the link somewhere, it becomes broken because linking stops at the first space. See for example here:

http://hapijs.com/resources#Gists and code samples
http://hapijs.com/resources#Projects built with hapi.js

**Comment by Marsup** [comment]
Indeed Chrome seems to be dumb about it, Firefox restores the %20 when you copy it.

**Comment by nlf** [comment]
closed via #411
